.travis.yml

```diff
@@ -4,18 +4,18 @@ node_js:
 - 6
 - 8
 before_install:
-- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ] && 
+- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ] &&
   openssl aes-256-cbc -K $encrypted_ac3aacad7ba8_key
   -iv $encrypted_ac3aacad7ba8_iv -in secrets.tar.enc -out test/resources/secrets.tar
   -d || true'
-- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ] 
+- '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ ! -z ${encrypted_ac3aacad7ba8_key} ]
   && cd test/resources/ && tar xvf secrets.tar
   && cd ../.. || true'
 - npm install -g typescript
 script:
 - tsc
 - npm run test-unit-travis || travis_terminate 1
-- npm run test-integration-travis || node scripts/report_integration_test.js || (cat test-output.log && travis_terminate 1)
```

Comment:
I'm good with this but wondering what your motives were for removing the output logging

Reply:
I added it previously cuz i was parsing stdout. now that i'm using json, the print is just ugly. Forgot to remove it with the previous PR